bindings/rascal/utils/fps.py

```diff
@@ -87,3 +93,194 @@ def fps(feature_matrix, n_select, starting_index=None,
     return_dict["fps_hausdorff_d2"] = lmin_d2
 
     return return_dict
+
+
+class FPSFilter(BaseIO):
```

Comment:
This looks very nice. Does it basically just work as a wrapper around the existing `fps` function?

Reply:
That is what it does yes. It follows the same spirit as in cur.py